languages/en.ini

```diff
@@ -1184,7 +1184,13 @@ See also = "See also"
 see_all_ellipsis = "see all…"
 Select this record = "Select this record"
 Select your carrier = "Select your carrier"
+select_all_on_page = "Select all items on the page"
+select_item_checked_out_renew = "Select item for renewing"
+select_item_hold_cancel = "Select item for canceling holds or recalls"
```

Comment:
This won't work if updating of holds is enabled.

Reply:
@elsenhans, I'm not sure if "edit" fully encompasses "cancel."

Do we need to be specific about the action? Can we just say something like "select hold/recall for bulk action"? Or just "select hold/recall"?

Reply:
Right, correct would be "_Select item for updating or canceling holds or recalls_", but also too long. 
What do you think @ckaz ?

Reply:
The other problem is that not every ILS supports both updating and canceling holds, and figuring out exactly which message is correct would be complex. That's why I favor saying less specific things, so the language will not be misleading.

Reply:
What's available also depends on the hold status within an ILS, which is why I also would keep it pretty generic. Also "item" is a potentially confusing term. I'd rather select a hold or recall than item. A more verbose but still pretty generic text could be e.g. "_Select hold or recall for further action_".

Reply:
Yes, I could live with "Select hold or recall for further action." Thanks, @EreMaijala!